packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -4402,6 +4402,111 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /reversePurposes:
+    post:
+      tags:
+        - purposes
+      summary: Create a new e-service Purpose
+      description: Create a new purpose for an e-service in receive mode
```

very minor, but consistent with other references in the file
```suggestion
      summary: Create a new E-Service Purpose
      description: Create a new purpose for an E-Service in receive mode
```